<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Raspis</class>
 <widget class="QDialog" name="Raspis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1274</width>
    <height>794</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#Raspis {
	background-color: rgb(187, 245, 243);
}</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="95,5">
   <item row="0" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <property name="styleSheet">
      <string notr="true">QHeaderView::section:horizontal { 
color: rgb(0, 0, 0);
font: bold;
padding-bottom: 2px;
background-color: rgb(0, 230, 245);
}</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="15,10,10,10,10,25,20">
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: rgb(0, 230, 245);
font:bold;
border: 1px solid #000000;
}
QPushButton::hover {
	background-color: rgb(0, 243, 255);
}
QPushButton::pressed {
	border: 3px solid #000000;
}</string>
       </property>
       <property name="text">
        <string>Очистить фильтр</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
     width: 10px;
	background-color:  rgb(0, 230, 245);
    image: url(:/new/prefix1/1.png);
    padding-left: 7px;
	padding-right: 7px;

    border-left-width: 1px;
    border-left-color: black;
    border-left-style: solid;  
 }
QComboBox {
background-color: rgb(255, 255, 255);
border: 1px solid black;
}
QComboBox QAbstractItemView {
    background-color: rgb(255, 255, 255);
 }</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: rgb(0, 230, 245);
font:bold;
border: 1px solid #000000;
}
QPushButton::hover {
	background-color: rgb(0, 243, 255);
}
QPushButton::pressed {
	border: 3px solid #000000;
}</string>
       </property>
       <property name="text">
        <string>Искать по типу</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QDateEdit {
background-color: rgb(255, 255, 255);
border: 1px solid #000000;
}
QDateEdit::up-button {
     background-color: rgb(0, 230, 245);
 }
QDateEdit::up-button:hover {
     background-color: rgb(0, 243, 255);
 }
QDateEdit::down-button {
     background-color: rgb(0, 230, 245);
 }
QDateEdit::down-button:hover {
     background-color: rgb(0, 243, 255);
 }
QDateEdit::up-arrow {
    image: url(:/new/prefix1/11.png);
     width: 7px;
     height: 7px;
 }
QDateEdit::down-arrow {
    image: url(:/new/prefix1/1.png);
     width: 7px;
     height: 7px;
 }
QDateEdit::down-button:pressed {
	border: 3px solid #000000;
}
QDateEdit::up-button:pressed {
	border: 3px solid #000000;
}


</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: rgb(0, 230, 245);
font:bold;
border: 1px solid #000000;
}
QPushButton::hover {
	background-color: rgb(0, 243, 255);
}
QPushButton::pressed {
	border: 3px solid #000000;
}</string>
       </property>
       <property name="text">
        <string>Искать по дате
отправления</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: rgb(0, 230, 245);
font:bold;
border: 1px solid #000000;
}
QPushButton::hover {
	background-color: rgb(0, 243, 255);
}
QPushButton::pressed {
	border: 3px solid #000000;
}</string>
       </property>
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
